packages/drivers/local-driver/src/test/types/validateLocalDriverPrevious.generated.ts

```diff
@@ -4,7 +4,7 @@
  */
 /*
  * THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
- * Generated by fluid-type-validator in @fluidframework/build-tools.
+ * Generated by fluid-type-test-generator in @fluidframework/build-tools.
```

Comment:
Note there are a few other hits for "Generated by fluid-type-validator" (as opposed to "Generated by fluid-type-test-generator") in the repo currently.  Some will go away in `next` as packages are removed or hidden, but looks like there are a few packages that will still need updated typetest scripts at least.